cfDNApipe/cfDNA_utils.py

```diff
@@ -1431,17 +1633,23 @@ def stdRes(res, d, H):
     return res_std
 
 
-def RobustSVR(reference, mixtureData, iter_num=10, confidence=0.75, w_thresh=10, unknown=False):
+def RobustSVR(
+    reference, mixtureData, iter_num=10, confidence=0.75, w_thresh=10, unknown=False
+):
     tissueNum = np.size(reference, 1)
     numOfSamples = np.size(mixtureData, 1)
     markerNumber = np.size(reference, 0)
     proportionDeconvolution = (
-        np.zeros([tissueNum, numOfSamples]) if unknown == False else np.zeros([tissueNum + 1, numOfSamples])
+        np.zeros([tissueNum, numOfSamples])
+        if unknown == False
+        else np.zeros([tissueNum + 1, numOfSamples])
     )
     for i in range(numOfSamples):
         iterReference = reference
         itermixtureData = mixtureData[:, i]
-        mixture = sm.RLM(itermixtureData, iterReference).fit()
+        mixture = sm.RLM(
```

Comment:
@saranouri This update is required because Hubert weight function results in division by zero errors in RLM when there are extreme outliers. Tukay seems to work because it is a more smooth function. 

Documentation for both model and weight functions is here: https://www.statsmodels.org/devel/rlm.html